bootleg/run.py

```diff
@@ -99,7 +99,10 @@ def main_worker(gpu, args, mode, world_size):
         # whether the machine is responsible for removing/writing files on the CPU
         is_writer = (rank % args.run_config.ngpus_per_node) == 0
     if torch.cuda.is_available() and not args.run_config.cpu:
+        logger.info('Running Inference on GPU {}'.format(args.run_config.gpu))
```

Comment:
My very small nit is that inference applies eval while this is printed for all runs. Can you change the word Inference to be like "Model" or just remove it?

Reply:
I added it for debugging purposes so yeah I can remove it.